apps/agent-service/src/repositories/agent-service.repository.ts

```diff
@@ -155,6 +155,58 @@ export class AgentServiceRepository {
         }
     }
 
+    /**
+     * Store DID details
+     * @param storeDidDetails
+     * @returns did details
+     */
+    // eslint-disable-next-line camelcase
+    async storeDidDetails(storeDidDetails: StoreDidDetails): Promise<org_dids> {
+        try {
+          const {orgId, did, didDocument, isPrimaryDid, userId, orgAgentId} = storeDidDetails;
+
+          return this.prisma.org_dids.create({
+                data: {
+                    orgId,
+                    did,
+                    didDocument,
+                    isPrimaryDid,
+                    createdBy: userId,
+                    lastChangedBy: userId,
+                    orgAgentId
+                }
+            });
+        } catch (error) {
+            this.logger.error(`[storeDidDetails] - Store DID details: ${JSON.stringify(error)}`);
+            throw error;
+        }
+    }
+
+
+    /**
+     * Set primary DID
+     * @param did
+     * @returns did details
+     */
+    // eslint-disable-next-line camelcase
+    async setPrimaryDid(isPrimaryDid:string, orgId:string): Promise<org_agents> {
+        try {
+          return await this.prisma.org_agents.update({
```

Comment:
Remove unnecessary 'await' in function call
```suggestion
          return this.prisma.org_agents.update({
```